ui/app/templates/permission_scheduled_change_update_modal.html

```diff
@@ -16,7 +16,9 @@ <h3 class="modal-title">  Edit Scheduled Change for {{ sc.username }}</h3>
       <div class="col-md-6" >
         <div class="form-group" ng-class="{'has-error': errors.permission}">
         <label for="id_permission">permission</label>
-        <input type="text" class="form-control" id="id_permission" ng-model="sc.permission">
+          <select name="singleSelect" class="form-control" id="id_permission" ng-model="sc.permission">
```

Comment:
This ng-model doesn't appear to be working. When I edit an existing scheduled permission change, the drop down always has "--- Please select ---" instead of the actually selecting the permission.

Reply:
@mozbhearsum Didn't notice it earlier. Fixed it now.